### PR TITLE
Update Rise of Nations Extended Edition

### PR DESCRIPTION
**Pull request by LukeShortCloud**
to include DirectPlay for more reliable multiplayer connections.

### CODE REVIEW

**Comment by LukeShortCloud** [comment]
I have not personally tested this yet but I have seen many reports of this helping. The latest report being from 1 month ago.

https://www.protondb.com/app/287450

On vanilla Proton, game matches will randomly disconnect and then crash about 1 hour into the game. That is a problem when games can last many hours. My last match took 2.5 hours.

Luckily, this game has a unique multiplayer save feature. My friends and I have been using that as a short-term workaround to save and reload games when they crash (the person whose game did not crash is still able to save).

File: gamefixes-steam/287450.py [hunk 1]
```diff
@@ -25,5 +25,16 @@ def main():
...
+    util.protontricks('directplay')
+    util.winedll_override('dplaysvr.exe', 'n')
+    util.winedll_override('dplayx', 'n')
+    util.winedll_override('dpmodemx', 'n')
+    util.winedll_override('dpnet', 'n')
+    util.winedll_override('dpnhpast', 'n')
+    util.winedll_override('dpnhupnp', 'n')
+    util.winedll_override('dpnsvr.exe', 'n')
+    util.winedll_override('dpwsockx', 'n')
+
```
Comment (R1kaB3rN) [inline gamefixes-steam/287450.py:38]: I believe lines 30-37 may be redundant here as specifying the `directplay` verb already applies the native overrides for those files.

**Comment by Root-Core** [comment]
I'm not sure this fix is necessary. [Others say](https://github.com/ValveSoftware/Proton/issues/298#issuecomment-506982376) that `directmusic` and `sound=alsa` are sufficient. It would be nice if someone could give feedback on the current fix.

**Comment by LukeShortCloud** [comment]
Most people are not playing multiplayer. `directmusic` and `sound=alsa` only affects being about to hear the music and sound affects throughout the game.

There are 2 reports in the GitHub Issue and 15 reports in the ProtonDB page that `directplay` is needed for online play. I am able to play online with no workarounds but me and my other friends on Linux constantly run into random connection issues. I believe DirectPlay will fix that.

I will test this out with my friends next time we play online and report back (by applying `protontricks directplay` to everyone's game).

**Pull request merged**